### DISCUSSION TITLE
Is it possible to disable autosave in the builder and require users to manually save?

### DISCUSSION THREAD

**Discussion by djkunkel** [vueform/builder#3] [Questions] [answered]
I see in the config you can change the delay before autosave occurs, but is is possible to make saving a manual operation? I would only like to save the form state to the backend when the user explicitly wants to.

**Comment by adamberecz** [comment, marked as answer]
The built-in save function will only save the changes to the local storage and trigger the `@save` event. You can add a **Save** button somewhere outside the builder that triggers the actual saving. You can reach the builder object with `builder$.builder` after assigning a ref to `<VueformBuilder ref="builder$">`.
